diff --git a/dist/oauth2/validate-authorization-code.mjs b/dist/oauth2/validate-authorization-code.mjs
index e36749ffd563e5c296c2d765e7fae45d329b37e6..85f5ee65e15a831b53f69a3d468bb669be22ed48 100644
--- a/dist/oauth2/validate-authorization-code.mjs
+++ b/dist/oauth2/validate-authorization-code.mjs
@@ -1,7 +1,7 @@
 import { NO_FOLLOW_REDIRECT, assertResponseNotRedirect, fetchRefusingRedirects } from "./reject-redirects.mjs";
 import { getOAuth2Tokens } from "./utils.mjs";
 import { base64 } from "@better-auth/utils/base64";
-import { createRemoteJWKSet, customFetch, jwtVerify } from "jose";
+import { createRemoteJWKSet, jwtVerify } from "jose";
 //#region src/oauth2/validate-authorization-code.ts
 async function authorizationCodeRequest({ code, codeVerifier, redirectURI, options, authentication, deviceId, headers, additionalParams = {}, resource }) {
 	options = typeof options === "function" ? await options() : options;
@@ -70,7 +70,7 @@ async function validateAuthorizationCode({ code, codeVerifier, redirectURI, opti
 	return getOAuth2Tokens(data);
 }
 async function validateToken(token, jwksEndpoint, options) {
-	return await jwtVerify(token, createRemoteJWKSet(new URL(jwksEndpoint), { [customFetch]: async (url, init) => {
+	return await jwtVerify(token, createRemoteJWKSet(new URL(jwksEndpoint), { fetch: async (url, init) => {
 		const response = await fetch(url, {
 			...init,
 			...NO_FOLLOW_REDIRECT
